### PR TITLE
Restore the 'server_stats' and 'client_stats' for legacy repl

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

File: src/riak_repl_console.erl [hunk 1]
```diff
@@ -621,8 +617,9 @@ server_stats() ->
         undefined ->
             [{sources, RT2}];
         _ ->
-            [{sources, rpc:call(LeaderNode, ?MODULE, server_stats_rpc,
-                                     [])++RT2}]
```
Comment (bookshelfdave) [inline src/riak_repl_console.erl:625]: won't this break for customers that are already scraping BNW repl stats?

**Comment by bookshelfdave** [comment]
Both client and server can be testing with:
curl -q http://localhost:port/riak-repl/stats

File: src/riak_repl_wm_stats.erl [hunk 2]
```diff
@@ -143,4 +165,44 @@ jsonify_stats_test_() ->
       ?assertEqual(Expected, jsonify_stats(Stats, []))
     end}].
 
+jsonify_stats_test_2() ->
+    [{"legacy server_stats and client_stats", fun() ->
```
Comment (Vagabond) [inline src/riak_repl_wm_stats.erl]: The tuple here should just be added to the list inside jsonify_stats_test_()

**Comment by Vagabond** [comment]
Closed in favor of PR against 1.3 branch.

**Pull request closed without merging**